### PR TITLE
Update ru.wlist

### PR DESCRIPTION
**Pull request by MrZomka**
Added plural and declension words and a few other words that I forgot to add to the list.

### CODE REVIEW

File: src/main/resources/filters/ru.wlist [hunk 1]
```diff
@@ -2,34 +2,63 @@ blyad
 blyadt
 cyka
 блядь
+бля
+блять
```
Comment (rtm516) [inline src/main/resources/filters/ru.wlist:6]:
```suggestion
бля((д|т)ь)?
```

File: src/main/resources/filters/ru.wlist [hunk 2]
```diff
@@ -2,34 +2,63 @@ blyad
...
 сука
 сучка
 суки
+суку
 сучки
+сучку
```
Comment (rtm516) [inline src/main/resources/filters/ru.wlist:12]:
```suggestion
суч?к(и|у)
```

File: src/main/resources/filters/ru.wlist [hunk 3]
```diff
@@ -2,34 +2,63 @@ blyad
...
 хуй
 хуя
+хуям
```
Comment (rtm516) [inline src/main/resources/filters/ru.wlist:15]:
```suggestion
хуям?
```

File: src/main/resources/filters/ru.wlist [hunk 4]
```diff
@@ -41,3 +70,9 @@ cyka
 ебнутный
 ебануться
 залупа
+залупе
+залупы
+ёпта
+епта
+ёпты
+епты
```
Comment (rtm516) [inline src/main/resources/filters/ru.wlist:78]:
```suggestion
(ё|е)пт(а|ы)
```

File: src/main/resources/filters/ru.wlist [hunk 5]
```diff
@@ -2,34 +2,63 @@ blyad
...
 пидорасина
+пидорасину
 пидр
+пидры
+пидру
+пидрам
 пизда
+пизды
+пизде
+пизду
```
Comment (rtm516) [inline src/main/resources/filters/ru.wlist:57]:
```suggestion
пизд(а|е|ы|у)
```

File: src/main/resources/filters/ru.wlist [hunk 6]
```diff
@@ -2,34 +2,63 @@ blyad
...
 пздц
 пиздец
+пиздецу
+пиздеца
```
Comment (rtm516) [inline src/main/resources/filters/ru.wlist:61]:
```suggestion
пиздец(у|а)?
```

File: src/main/resources/filters/ru.wlist [hunk 7]
```diff
@@ -2,34 +2,63 @@ blyad
...
+пидорам
 пидора
 пидорас
+пидорасу
 пидорасина
+пидорасину
 пидр
+пидры
+пидру
+пидрам
```
Comment (rtm516) [inline src/main/resources/filters/ru.wlist:53]:
```suggestion
пидр(у|ы|ам)?
```

File: src/main/resources/filters/ru.wlist [hunk 8]
```diff
@@ -2,34 +2,63 @@ blyad
...
+хуепроебины
 пидор
+пидоры
+пидору
+пидорам
 пидора
 пидорас
+пидорасу
 пидорасина
+пидорасину
```
Comment (rtm516) [inline src/main/resources/filters/ru.wlist:49]:
```suggestion
пидорасин(у|а)
```

File: src/main/resources/filters/ru.wlist [hunk 9]
```diff
@@ -2,34 +2,63 @@ blyad
...
 сучка
 суки
+суку
 сучки
+сучку
 хуй
 хуя
+хуям
 хуи
+хую
```
Comment (rtm516) [inline src/main/resources/filters/ru.wlist:17]:
```suggestion
ху(и|ю)
```

File: src/main/resources/filters/ru.wlist [hunk 10]
```diff
@@ -2,34 +2,63 @@ blyad
...
 хер
+херу
 хера
+херам
```
Comment (rtm516) [inline src/main/resources/filters/ru.wlist:21]:
```suggestion
хер(у|ам|а)?
```

File: src/main/resources/filters/ru.wlist [hunk 11]
```diff
@@ -2,34 +2,63 @@ blyad
...
 херы
 нахуй
 нахуя
 нахер
 нахера
 ахуел
+ахуели
 ахуела
```
Comment (rtm516) [inline src/main/resources/filters/ru.wlist:29]:
```suggestion
ахуел(и|а)?
```

File: src/main/resources/filters/ru.wlist [hunk 12]
```diff
@@ -2,34 +2,63 @@ blyad
...
 ахерел
+ахерели
 ахерела
```
Comment (rtm516) [inline src/main/resources/filters/ru.wlist:32]:
```suggestion
ахерел(и|а)?
```

File: src/main/resources/filters/ru.wlist [hunk 13]
```diff
@@ -2,34 +2,63 @@ blyad
...
 хуепроёбина
+хуепроёбину
+хуепроёбине
+хуепроёбины
 хуепроебина
+хуепроебину
+хуепроебине
+хуепроебины
```
Comment (rtm516) [inline src/main/resources/filters/ru.wlist:40]:
```suggestion
хуепро(ё|е)бин(а|у|е|ы|а)?
```

File: src/main/resources/filters/ru.wlist [hunk 14]
```diff
@@ -2,34 +2,63 @@ blyad
...
 пидор
+пидоры
+пидору
+пидорам
```
Comment (rtm516) [inline src/main/resources/filters/ru.wlist:44]:
```suggestion
пидор(у|ы|ам)?
```

File: src/main/resources/filters/ru.wlist [hunk 15]
```diff
@@ -2,34 +2,63 @@ blyad
...
 пидора
 пидорас
+пидорасу
```
Comment (rtm516) [inline src/main/resources/filters/ru.wlist:47]:
```suggestion
пидора(су|с)?
```

**Review by rtm516 — CHANGES_REQUESTED** [review]

**Review by MrZomka — COMMENTED** [review]
I did not know you can do that xD

**Review by rtm516 — APPROVED** [review]

**Pull request merged**